.eslintrc.json

```diff
@@ -57,7 +57,10 @@
     "jsdoc/require-returns-description": "error",
     "jsdoc/check-alignment": "error",
     "jsdoc/check-param-names": "error",
-    "react-hooks/exhaustive-deps": "error"
+    "react-hooks/exhaustive-deps": "error",
+    "sonarjs/todo-tag": "warn",
+    "sonarjs/redundant-type-aliases": "warn",
```

Comment:
@frano-m this is triggered for cases like the below in [app/apis/catalog/ncpi-catalog/common/entities.ts](https://github.com/DataBiosphere/data-browser/blob/72f9cf377f41d1221f5936fd98a638eba9167885/app/apis/catalog/ncpi-catalog/common/entities.ts#L13C1-L13C30). Not sure if it should be disabled, kept as a warn, and/or if we should remove those cases, do you have thoughts?

```
export type DbGapId = string;
```